src/main/java/com/cburch/logisim/fpga/download/XilinxDownload.java

```diff
@@ -146,7 +147,7 @@ public ProcessBuilder PerformStep(int stage) {
 
   @Override
   public boolean readyForDownload() {
-    return new File(SandboxPath + ToplevelHDLGeneratorFactory.FPGAToplevelName + "." + BitfileExt).exists();
+    return new File(SandboxPath + ToplevelHDLGeneratorFactory.FPGAToplevelName + "." + bitfileExt).exists();
```

Comment:
sandboxPath instead of SandboxPath ?

Reply:
I leave `SandboxPath` unaltered. It is used in multiple places in that file, so changing it would increase the diff